guide/src/tutorials/production/start-relaying.md

```diff
@@ -0,0 +1,40 @@
+# Start relaying
+
+In section [Setup Grafana](./setup-grafana.md), you did set up a Grafana dashboard which is now waiting to receive data produced by `hermes` and running on port 3000. You also configured Hermes in section [Setup Hermes](./setup-hermes.md) and added the keys you will be using. 
+
+---
+
+## Create an empty log file
+
+Promtail is shipping every log file from `/var/log` to Loki. Follow the steps below to create an empty log file for Hermes:
+```shell
+sudo touch /var/log/hermes.log 
+sudo chown $(whoami) /var/log/hermes.log 
+```
+You should now have an empty `hermes.log` file that you can access and see on the Grafana Dashboard, on the `explore` page, if you select Loki as a data source. You can query the label `filename=hermes.log`.
+
+## Start relaying
+
+Follow the steps to get started :
+
+- Open your dashboard. Make sure it gets refreshed every 5s.
+
+- In a new terminal, run `hermes start &> hermes.log`. 
+
+If the command runs successfully, you should be able to see the metrics panels displaying data on the Grafana Dashboard and you should also be able to see the logs on the `Logs` panel at the top of the dashboard. You can also explore them on the `explore` page.
```

Comment:
Would it be possible to add images of the dashboard so that readers can get a better understanding of this?

Reply:
It is possible but I can't run the tutorial with Cosmoshub and Osmosis as I would need tokens from both chains. If I run it locally, it will display every metrics with a label `chain=ibc-0` and `chain=ibc-1` instead of `cosmoshub-4` and `osmosis-1`.